### DISCUSSION TITLE
Error deleting variable in Actions secrets and variables

### DISCUSSION THREAD

**Discussion by felicitasfigfag** [community/community#76842] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

<img width="473" alt="Screenshot 2023-11-24 at 11 41 49" src="https://github.com/community/community/assets/78109113/424413f8-e4cd-4476-8d8a-2d2a9c1eee8c">

The input field to write the password when trying to delete a Var in Settings > Security > Secrets and Variables > Actions > Variables seems to be unclickable.
It does work when trying to edit the value, but the input field is disabled when trying to delete the variable. To be specific: nothing shows when I write the password, and it does not select when I click on the field.

**Comment by hugofialho** [comment]
same here
